recipes/r-boot/build.sh

```diff
@@ -1,13 +0,0 @@
-#!/bin/bash
-
-# R refuses to build packages that mark themselves as Priority: Recommended
-mv DESCRIPTION DESCRIPTION.old
-grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
```

Comment:
Oh, sorry. I can only judge from the comment, which sounds like those lines are only necessary for those priority: recommended packages. Those are only a handful, that should shortly be in conda-forge (because I want to package them first and also port the r-recommended package from the r channel). Hence, I'd say the lines are not necessary for other packages.